style.css

```diff
@@ -0,0 +1,31 @@
+#main-doc {
+  height: 100px;
+  width: 100px;
+}
+
+#navbar {
+  position: fixed;
+  width: 250px;
+  height: 100px;
+  text-align: center;
+}
+
+#technical-doc-page {
+  width: 700px;
+  margin-left: 300px;
+}
+
+header {
```

Comment:
Same mistake, don't use header as a h1 tag. So don't stylize the header as a h1

Reply:
same here